### PR TITLE
commit for first release

### PR DESCRIPTION
**Pull request by kriii**
This branch is for the first release of `ain-py`.

After these codes are passed review, I'll be doing follows.

1. Separate `utils.py` from `ain-py` to make `ain-util-py`.
2. Release `ain-util-py`.
3. Correct the `ain-py`.
4. Release `ain-py`.
5. Complete merge.

### CODE REVIEW

File: ain/ain.py [hunk 1]
```diff
@@ -1,38 +1,219 @@
...
+        """
+        Sets a new provider
+        """
+        self.provider = Provider(self, providerUrl)
+        self.chainId = chainId
+        self.net = Network(self.provider)
+        self.wallet = Wallet(self, self.chainId)
+        self.db = Database(self, self.provider)
+
+    # TODO(kriii) : return objects typing
```
Comment (platfowner) [inline ain/ain.py:33]: minor: could you remove space between ')' and ':'? and period '.' at the end?

File: ain/utils.py [hunk 2]
```diff
@@ -66,37 +125,162 @@ def toChecksumAddress(address: str) -> str:
...
+    """
+    Returns the public key of a given private key.
+    
+    args:
+        privateKey (bytes): A private key must be 256 bits wide
+    """
+    prK = PrivateKey(privateKey)
+    return prK.public_key.format(False)[1:]
+
+# TODO(kriii): support isSEC1
```
Comment (platfowner) [inline ain/utils.py:266]: minor: could you add period '.' and the end?

**Comment by kriii** [comment]
Fixed TODOs. PTAL

File: ain/ain.py [hunk 3]
```diff
@@ -1,38 +1,219 @@
...
+        """
+        return await self.provider.send(
+            "ain_sendSignedTransaction", {"signature": signature, "tx_body": txBody}
+        )
+
+    async def sendTransactionBatch(self):
+        """
+        Sends signed transactions to the network.
+        """
+        pass
```
Comment (liayoo) [inline ain/ain.py:143]: could you leave TODOs for the functions that haven't been implemented?

**Comment by liayoo** [comment]
As a next step (after the 1-5), when you get a chance, could you add simple tests & describe how to run them in README.md?

**Comment by kriii** [comment]
> As a next step (after the 1-5), when you get a chance, could you add simple tests & describe how to run them in README.md?

Sure!

File: ain/ain.py (hunk 3 above)
Comment (kriii) [reply ain/ain.py:144]: Added. PTAL

File: ain/ain.py [hunk 4]
```diff
@@ -1,38 +1,223 @@
...
+
+    async def sendSignedTransaction(self, signature: str, txBody: TransactionBody):
+        """
+        Sends a signed transaction to the network
+        """
+        return await self.provider.send(
+            "ain_sendSignedTransaction", {"signature": signature, "tx_body": txBody}
+        )
+
+    # TODO(kriii): implement this function
```
Comment (platfowner) [inline ain/ain.py:139]: could you add a period at the end of all todos as well?

**Review by platfowner — COMMENTED** [review]
LGTM with some minor comments.
Please wait for @liayoo 's approval.
Thanks!

**Review by liayoo — APPROVED** [review]
LGTM, thanks for the nice implementation!

**Comment by kriii** [comment]
Merged! Thanks for the review!

**Pull request merged**